python/hsfs/feature_view.py

```diff
@@ -56,9 +54,11 @@
 from hsfs.core.vector_db_client import VectorDbClient
 from hsfs.decorators import typechecked
 from hsfs.feature import Feature
+from hsfs.hopsworks_udf import HopsworksUdf
 from hsfs.statistics import Statistics
 from hsfs.statistics_config import StatisticsConfig
 from hsfs.training_dataset_split import TrainingDatasetSplit
+from hsfs.transformation_function import TransformationFunction
```

Comment:
will be nice to add a property which return the list of feature name after transformation

Reply:
added property `output_column_names`

Reply:
I mean `fv.transformed_features`, maybe a better name, should return the list of feature name you will get in `get_feature_vector`

Reply:
Renamed property that returns list of features names in a feature view after transformation to `transformed_features` .

Reply:
Added property transformed_features in feature view to get name of features after transformation functions have been applied.